pyrightconfig.testcases.json

```diff
@@ -6,12 +6,21 @@
         "stubs/**/@tests/test_cases"
     ],
     "typeCheckingMode": "strict",
+    // Extra strict settings
+    "reportShadowedImports": "error", // Don't accidentally name a file something that shadows stdlib
+    "reportCallInDefaultInitializer": "error",
```

Comment:
Not sure it makes sense to have this enabled for our test-case files. It's often not great practice in Python code, but we probably wouldn't include this (anti-)pattern in a test case unless we had a reason to do so, in which case this lint would be annoying. flake8-bugbear also already checks for this in our `.py` files anyway.

Reply:
I think this makes sense given `"enableTypeIgnoreComments": true` and the associated comment.